core/src/main/java/org/lflang/target/property/DockerProperty.java

```diff
@@ -168,7 +174,8 @@ public enum DockerOption implements DictionaryElement {
     RTI_IMAGE("rti-image", PrimitiveType.STRING),
     PRE_BUILD_SCRIPT("pre-build-script", PrimitiveType.STRING),
     PRE_RUN_SCRIPT("pre-run-script", PrimitiveType.STRING),
-    POST_BUILD_SCRIPT("post-build-script", PrimitiveType.STRING);
+    POST_BUILD_SCRIPT("post-build-script", PrimitiveType.STRING),
+    DOCKER_CONFIG_FILE("docker-config-file", PrimitiveType.STRING);
```

Comment:
Why not call it `docker-compose-override`? I think that's more specific.
```suggestion
    DOCKER_CONFIG_FILE("docker-compose-override", PrimitiveType.STRING);
```

Reply:
Could even call it `compose-override` because it's already in the `docker` dictionary...

Reply:
Makes sense. Should I change the key to "compose-override-file" then? Or perhaps just "compose-file" might suffice too because from the user perspective this is nearly identical to writing a docker-compose file for the reactors.

Reply:
Not sure. Let me tag @cmnrd and @elgeeko1 to ask for their opinion.